admin/preprint_providers/views.py

```diff
@@ -249,7 +248,7 @@ def get_page_provider(self):
 
     def add_subjects(self, provider, subject_data):
         from osf.management.commands.populate_custom_taxonomies import migrate
-        migrate(provider._id, subject_data)
+        migrate(provider=provider._id, data=subject_data)
```

Comment:
Yes, github, those were definitely the things that changed and the things that stayed the same.

Reply:
lol